Dockerfile

```diff
@@ -0,0 +1,13 @@
+ARG GO_VERSION=1.13.1
+FROM golang:${GO_VERSION}-alpine
+
+ARG TSBS_VERSION=master
+RUN apk update && apk add --no-cache --virtual .build-deps git \
+    && mkdir -p ${GOPATH}/src/github.com/timescale/ \
```

Comment:
we're using go modules, no need to clone into $GOPATH and to do go get
so we can remove two lines (mkdir line 6, and go get line 10)

Reply:
also, I would suggest using multi-stage build (https://docs.docker.com/develop/develop-images/multistage-build/), so you don't need to worry about deleting the source and apk del. You can just copy the built binary

Reply:
Yeah I agree with all of this.

And now `1.13.3` is the latest